CONTRBUTING.md

```diff
@@ -0,0 +1,22 @@
+# How to contribute
+We love pull requests from everyone. Please take a look at the contributing
+guidelines below before making a pull request.
+
+## Getting started
+Make sure you have cloned the project under `$GOPATH/src` by using the `go get` command.
```

Comment:
Maybe add the instructions to build the binary and shared libs here, to be a little bit more complete?
You can find the commands used in the travis.yml files.

Reply:
```
pushd cmd/ubuntu-report && go build -o ../../build/ubuntu-report && popd
go generate ./...
```
these?

Reply:
You don't need pushd/pop, just cd, and `go build`, without controling the main directory.

Then, for go generate, this is another section, and list the generated assets (completion files, man pages and shared library).

I would add as well a "how to generate the shared libs". Look for "go:generate" inside the code files (grep for it ;))  and try to simplify the go build command for people running it into the desired category.

I hope this makes sense :)